l3kernel/l3keys.dtx

```diff
@@ -3249,6 +3358,7 @@
       { \prg_return_true: }
       { \prg_return_false: }
   }
+\prg_generate_conditional_variant:Nnn \keys_if_exist:nn { nx } { T , F , TF }
```

Comment:
Should be `ne` as variant instead of `nx`, imho. `\keys_if_exist:nn` is expandable, and even if the kernel does not yet strictly require `\expanded` it is in practice almost always available.

Reply:
Well ... that's complicated as they handle `#` differently, so we can never drop `x`-type even where the underlying code is expandable. I'll think about this one.

Reply:
But that's a new addition so you can pick the one you want now. And the handling of data which should be turned into a string is more natural using `e`-type expansion, imho, because of the `#` handling.